README.md

```diff
@@ -86,6 +86,20 @@ $databases['default']['default'] = array (
 );
 ```
 
+### Memcached settings
+
+To enable memcached add these settings to your `settings.php` file.
+Replace `[PATH TO MEMCACHE MODULE]` with path to memcached module in your project.
```

Comment:
To stick with the same style, please update as follow:

Replace `</path/to/memcache-module>` with path to [memcache module](https://www.drupal.org/project/memcache) in your project. E.g. `sites/all/modules/contrib/memcache`